### ISSUE TITLE
Removing slds wrapper and prefixes

### ISSUE THREAD

**Issue by alanpilloud** [salesforce-ux/design-system#21]
Hello,

As I'd like to use only one framework and that collisions will not be possible, I'm wondering if there is a way to remove the slds body class and the slds- class suffix ?

Have a nice day.

**Comment by Guria** [comment]
changing https://github.com/salesforce-ux/design-system/blob/winter-16/ui/dependencies/namespace.scss should do the trick

**Comment by alanpilloud** [comment]
Thanks a lot ! That did the trick !

**Comment by amyleesalesforce** [comment]
So while you CAN do that, we don't recommend removing prefixes. They are a powerful way of ensuring you are using the current, exact version of the Design System CSS. We are also concerned about the long-term. (You know, enterprise CSS.)

I guess if it is an isolated project then go for it.

**Comment by alanpilloud** [comment]
So you mean that the .slds- prefixes might change over the time ? Well, it's not that prefix that bothered me the most but more the body class which adds complexity to css selectors.

Thanks a lot for your responses which made me aware of each possibilities/reasons.

**Comment by stefsullrew** [comment]
@alanpilloud Are you referring to the scoped version of the file? (The ones that surround all the CSS with  the class `.slds`?)

If so, you only need to use a scoped version of the CSS if you will not have full control of the DOM. ie: building in Lightning Components, building something that will be added to Visualforce, mixing our framework with another framework (like Bootstrap, etc).

If you have full control, you do not need to scope your application. Using the namespace is up to you. (By namespace, I mean the `.slds-` that each selector begins with.

**Comment by alanpilloud** [comment]
I was referring to both usages, namespace and scoped classes. 

Here's what I did : 
 - I got rid of the `.slds` class to keep CSS complexity as lean as possible
 - I kept the `.slds-` prefix to separate slds classes from the few custom classes I use. This let me know directly if the class comes from slds or not.

I'm building a web application that I want to be as lean as possible, so I have full control on CSS since I limit myself to use only one framework and just a few custom CSS rules.

Thanks for your suggestions.
